engine/lib/base_jobmaster.go

```diff
@@ -63,6 +64,23 @@ type BaseJobMaster interface {
 	IsBaseJobMaster()
 }
 
+// BaseJobMasterExt extends BaseJobMaster with some extra methods.
+// These methods are used by framework and are not visible to JobMasterImpl.
+type BaseJobMasterExt interface {
+	// OnOpenAPIInitialized is used to initialize the openapi handler.
```

Comment:
How about using a name for action, like 'TriggerOpenAPIInitialize'.

Reply:
Done.